sdcm/cluster.py

```diff
@@ -4039,6 +4062,20 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
                 raise Exception("There is no pre-installed ScyllaDB")
 
         if not Setup.REUSE_CLUSTER:
+            # prepare and start saslauthd service
+            setup_script = dedent(f"""
+                sudo yum install -y cyrus-sasl
+                sudo systemctl enable saslauthd
+                echo 'MECH=ldap' | sudo tee -a /etc/sysconfig/saslauthd
+                sudo touch /etc/saslauthd.conf
+                echo 'saslauthd_socket_path: /run/saslauthd/mux' | sudo tee -a /etc/scylla/scylla.yaml
+            """)
```

Comment:
This is a bug #3396 .
We shouldn't ever assume installing by yum as it break support in all other platforms.
Please send a fix ASAP.

Reply:
Another bug #3395.